app/src/test/java/DataBinderMapperImpl.java

```diff
@@ -1,6 +1,13 @@
 package androidx.databinding;
 
-public class DataBinderMapperImpl extends MergedDataBinderMapper {
+/**
+ * Generated code necessary to run tests with Bazel.
+ *
+ * Typically, this code is generated by Bazel and included in the srcs list when running tests.
+ * However, the `enable_data_binding` flag somehow affects this process. For now, this file must be
+ * included in the srcs list of any Bazel test in app module.
+ */
+public final class DataBinderMapperImpl extends MergedDataBinderMapper {
```

Comment:
Please add TODO to remove on issue tracking making android_local_test work with data-binding.

Reply:
done